src/java/org/apache/cassandra/db/commitlog/CommitLogReader.java

```diff
@@ -120,14 +120,24 @@ static List<File> filterCommitLogFiles(File[] toFilter)
     /**
      * Reads all passed in files with minPosition, no start, and no mutation limit.
      */
-    public void readAllFiles(CommitLogReadHandler handler, File[] files, CommitLogPosition minPosition) throws IOException
-    {
+    public void readAllFiles(CommitLogReadHandler handler, File[] files, CommitLogPosition minPosition) throws IOException {
         List<File> filteredLogs = filterCommitLogFiles(files);
         int i = 0;
-        for (File file: filteredLogs)
-        {
+        for (File file : filteredLogs) {
             i++;
-            readCommitLogSegment(handler, file, minPosition, ALL_MUTATIONS, i == filteredLogs.size());
+            boolean success = false;
```

Comment:
what do you need `success` for? If you went without having `success`, then you would do `logger.info` call in `catch` block.

Also the way it is coded right now is that `handler.handleError` in catch block might throw RuntimeException you are not catching nor acting upon anywhere.